web/src/components/EmbeddedOffers.tsx

```diff
@@ -12,29 +12,29 @@ const Container = styled.div<{ withMargin: boolean }>`
   ${props => props.withMargin && 'margin-top: 48px;'}
 `
 
-type EmbeddedOfferProps = {
+type EmbeddedOffersProps = {
   category: CategoryModel
 } & CityRouteProps
 
-const EmbeddedOffers = ({ category, ...props }: EmbeddedOfferProps): ReactElement | null => {
+const EmbeddedOffer = ({ category, ...props }: EmbeddedOffersProps): ReactElement | null => {
```

Comment:
Wait, one became singular, one plural, I am confused

Reply:
That was the suggestion from @ztefanie and I think its okay. If you have some better naming idea let me know :)